modules/serverless-applications-checking-ingress-istio-routing.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/debugging-serverless-applications.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-applications-checking-ingress-istio-routing_{context}"]
+= Checking Ingress and Istio routing
+
+Sometimes, the Ingress and Istio routing have issues. You can see the details on them using the `oc` tool.
```

Comment:
Probably a hint here would be good to tell people that this is only relevant when istio is used as an ingress layer.